app/streaming/streaming.go

```diff
@@ -0,0 +1,9 @@
+package streaming
+
+// StreamServiceInitializers contains a map of predefined StreamServiceInitializer implementations
+import (
+	"github.com/provenance-io/provenance/internal/streaming"
+)
+
+// StreamServiceInitializers contains a map of supported StreamServiceInitializer implementations
+var StreamServiceInitializers = map[string]streaming.StreamServiceInitializer{}
```

Comment:
I'm guessing that this is something that's supposed to be added to during the `init` function for each plugin.

Do we not have any examples or plugins that we want available right now?

It'd be great if there were some unit/integration tests on this stuff too.

Reply:
I'll add a lightweight example that logs to `trace`.